src/main/java/cow/farm/repo/CowHerdService.java

```diff
@@ -0,0 +1,22 @@
+package cow.farm.repo;
+
+
+import cow.farm.utils.QuantityCheckingList;
+
+import java.util.HashMap;
+
+public class CowHerdService {
+
+	QuantityCheckingList<Cow> cowList = new QuantityCheckingList<>();
+
+	public Cow makeCow(Cow.Breed breed) {
+		Cow cow = new Cow(breed);
+		cowList.add(cow);
+		return cow;
+	}
+
+	public HashMap getHerdMap () {
```

Comment:
jeszcze jedna rzecz, HashMap to implementacja, zawsze to ale zawsze używaj interfejsów, jeżeli możesz, tutaj -> Map :)